.travis.yml

```diff
@@ -34,7 +34,7 @@ install:  # Install our test dependencies.
 
 script:  # Run our test suite.
   - if [[ -n $COVERAGE ]]; then export PYTEST_ADDOPTS="--cov=bidict --cov-config=.coveragerc"; fi
-  - python setup.py test
+  - pytest tests
```

Comment:
Should be just `pytest` as pytest.ini's `testpaths` [already specifies](https://github.com/jab/bidict/blob/master/pytest.ini#L2) the paths to test. I guess that's what was causing CI to fail for this PR. Fixed in 1f5a66c355d5b43dfa8bbb4c7f78cbf106e0f98b.

Reply:
Hmm wondering how this was working beforehand🤔

Reply:
By "beforehand" do you mean with "python setup.py test" or with "pytest tests"? With "python setup.py test", "test" refers to the "test" alias that was defined in setup.cfg, which was defined to just run a bare "pytest" command. With your "pytest tests", I think passing the "tests" path on the command line overrides the "testpaths" specified in pytest.ini, so the additional (doc)tests in the "docs" directory never got tested even though the tests in the "tests" dir ran and passed.

Reply:
I see makes sense. Thanks!